dwitter/models.py

```diff
@@ -24,7 +62,15 @@ class Comment(models.Model):
     posted = models.DateTimeField()
     reply_to = models.ForeignKey(Dweet, on_delete=models.CASCADE,
                                  related_name="comments")
-    author = models.ForeignKey(User, on_delete=models.CASCADE)
+    _author = models.ForeignKey(User, on_delete=models.CASCADE)
+
+    @property
+    def author(self):
+        return self._author
+
+    @author.setter
+    def author(self, value):
+        self._author = value
```

Comment:
The 500 error was caused by an error with the following message:

```
"_author" is an invalid keyword argument for this function.
```

I couldn't determine exactly why this was happening. Changing `Comment` to use `_author` instead of `author` seems to fix the problem.